openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/WheelchairAttributes.java

```diff
@@ -116,27 +120,53 @@ public void setSide(Side side) {
 		this.side = side;
 	}
 
-	public void setAttribute(Attribute attribute, int value) {
+	public boolean isSurfaceQualityKnown() {
+		return surfaceQualityKnown;
+	}
+
+	public void setSurfaceQualityKnown(boolean surfaceQualityKnown) {
+		this.surfaceQualityKnown = surfaceQualityKnown;
+	}
+
+	public boolean isSuitable() {
+		return suitable;
+	}
+
+	public void setSuitable(boolean suitable) {
+		this.suitable = suitable;
+	}
+
+	public void setAttribute(Attribute attribute, int value, boolean markSurfaceQualityKnown) {
```

Comment:
Thought about it, but would require major refactoring. It would also need to be involved in the planned implementation of flexible profiles, so I don't think this is a good time to address this.